src/api/app/controllers/webui/package_controller.rb

```diff
@@ -147,23 +146,14 @@ def dependency
   end
   # rubocop:enable Lint/NonLocalExitFromIterator
 
-  # TODO: bento_only
   def statistics
-    return if switch_to_webui2
```

Comment:
I think we still need this. not the `return` but the `switch_to_webui2`

Reply:
Done!